netlib/websockets/protocol.py

```diff
@@ -95,27 +95,25 @@ def server_handshake_headers(self, key):
         """
         return Headers(**{
             HEADER_WEBSOCKET_ACCEPT: self.create_server_nonce(key),
-            "Connection": "Upgrade",
-            "Upgrade": "websocket",
+            "connection": "Upgrade",
+            "upgrade": "websocket",
```

Comment:
Unrelated: Should we really use `HEADER_WEBSOCKET_ACCEPT` as constants here?
We never used constants for any of the other header keys...